bin/bios-baseline.sh

```diff
@@ -206,13 +206,13 @@ function run_ilo() {
         echo 'All NCNs are up-to-spec'
         return 0
     else
-        read -r -p "${#need_recon[@]} of $num_bmcs need BIOS Baseline applied ... proceed? [Y/n]:" response
+        read -r -p "${#need_recon[@]} of $num_bmcs need BIOS Baseline applied ... proceed? [y/N]:" response
```

Comment:
```suggestion
        read -r -p "${#need_recon[@]} of $num_bmcs need BIOS Baseline applied ... proceed? [y/n]:" response
```

Reply:
I would think the capital N makes more sense, since that is the default behavior, but I'm fine either way.